WooCommerce/Classes/ViewRelated/Products/Edit Product/DefaultProductFormTableViewModel.swift

```diff
@@ -416,6 +418,17 @@ private extension DefaultProductFormTableViewModel {
         return ProductFormSection.SettingsRow.WarningViewModel(icon: icon, title: title)
     }
 
+    func variationAttributesRow(isEditable: Bool) -> ProductFormSection.SettingsRow.ViewModel {
+        let icon = UIImage.customizeImage
+        let title = Localization.variationAttributesTitle
+        let details = "Any Color - Any Material"
+
+        return ProductFormSection.SettingsRow.ViewModel(icon: icon,
```

Comment:
Tip: Since you already declared the type you are going to return in the method signature, you can commit all the namespaces in favor of something like:
```swift
return .ViewModel(icon: icon, title: title, details: details, isActionable: isEditable)
```

Reply:
How do you decide between being more concise like this versus following the style used elsewhere?

It works when I return the init method like this:

```swift
return .init(icon: icon, title: title, details: details, isActionable: isEditable)
```

But since all the related methods include the full namespace I was following the same approach here. Is it better to be consistent with how it's done in other, related methods or go ahead with this more concise style?

Reply:
That's a tricky one! I usually always aim for consistency, however, I don't see this as a lack of consistency. I think of consistency more like similar approaches on how to achieve something. EG: Have this function as a static function would be a diff approach. 

Here I see removing the namespace as achieving the same but with less code, so I think we are good to go!